Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -180,6 +191,7 @@ private void Initialize ()
 				TranslatesAutoresizingMaskIntoConstraints = false
 			};
 			AddSubview (this.propertyList);
+			propertyFilter.NextKeyView = propertyList;
```

Comment:
for consistency, we should prefix both `propertyFilter` and `propertyList` with `this.`